src/main/java/core/selenium/utils/ReaderFile.java

```diff
@@ -0,0 +1,33 @@
+package core.selenium.utils;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * ReaderFile class.
```

Comment:
Please be more descriptions in your comment.

Reply:
Thank you, this wrong was fixed